pkg/reconciler/openshift/tektontrigger/extension.go

```diff
@@ -18,28 +18,71 @@ package tektontrigger
 
 import (
 	"context"
-
+	"github.com/go-logr/zapr"
+	mfc "github.com/manifestival/client-go-client"
 	mf "github.com/manifestival/manifestival"
 	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	"github.com/tektoncd/operator/pkg/client/clientset/versioned"
+	operatorclient "github.com/tektoncd/operator/pkg/client/injection/client"
 	"github.com/tektoncd/operator/pkg/reconciler/common"
+	occommon "github.com/tektoncd/operator/pkg/reconciler/openshift/common"
+	"go.uber.org/zap"
+	"knative.dev/pkg/injection"
+	"knative.dev/pkg/logging"
+	"os"
+	"path/filepath"
 )
 
 // NoPlatform "generates" a NilExtension
-func OpenShiftExtension(context.Context) common.Extension {
-	return openshiftExtension{}
+func OpenShiftExtension(ctx context.Context) common.Extension {
```

Comment:
Why do we need to do the same for triggers also?

Reply:
Because #190 did it as well :D CC: @vdemeester

Reply:
Same reason, I wanted it to be mounted in the triggers controller. Might not be required because triggers might not do any image digest resolving, … but I felt better safe than sorry.

Reply:
I see, we are calling `ApplyCABundle` here, so this is need for adding the volume mounts in triggers deployment. Perfect.